packages/react-provider/src/transports/types.ts

```diff
@@ -9,15 +9,16 @@ export interface IActionBlock {
   url: string;
   openInNewTab?: boolean;
 }
-export interface ICourierMessage {
-  event?: string;
-  type?: "message" | "event";
-  error?: string;
+
+export interface ICourierEventMessage {
+  event?: "read" | "unread" | "archive" | "mark-all-read";
```

Comment:
Why optional?

Reply:
i guess its not anymore but events used to not have this so its recently added.